app/commands/iteration/test_run/process.rb

```diff
@@ -63,8 +63,7 @@ def handle_error!
       end
 
       def cancel_other_services!
-        Iteration::Analysis::Cancel.(iteration.uuid)
-        Iteration::Representation::Cancel.(iteration.uuid)
+        ToolingJob::Cancel.(iteration.uuid)
```

Comment:
So, in this scenario, there are actually two jobs that need cancelling - the analysis job and the representation job. So I think we need to call "ToolingJob::Cancel" twice with the two different `job_type`s.